index.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/operator-framework/opm:latest
```

Comment:
included by mistake I guess, same as database/index.db file. Please remove

Reply:
yes, Thank you Michal. It was included by mistake. Removed.